acceptance/utilities_test.go

```diff
@@ -0,0 +1,154 @@
+	return "service-" + strconv.Itoa(int(time.Now().Nanosecond()))
+}
+
+func setupAndTargetOrg(org string) {
+	By("creating an org")
+
+	out, err := Carrier("create-org "+org, "")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	orgs, err := Carrier("orgs", "")
+	Expect(err).ToNot(HaveOccurred())
+	Expect(orgs).To(MatchRegexp(org))
+
+	By("targeting an org")
+
+	out, err = Carrier("target "+org, "")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	out, err = Carrier("target", "")
+	Expect(err).ToNot(HaveOccurred(), out)
+	Expect(out).To(MatchRegexp("Currently targeted organization: " + org))
+}
+
+func setupInClusterServices() {
+	out, err := Carrier("enable services-incluster", "")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	// Wait until plans appear
+	Eventually(func() error {
+		_, err = helpers.Kubectl("get clusterserviceclass mariadb")
+		return err
+	}, "5m").ShouldNot(HaveOccurred())
+}
+
+func makeApp(appName string) {
+	currentDir, err := os.Getwd()
+	Expect(err).ToNot(HaveOccurred())
+	appDir := path.Join(currentDir, "../sample-app")
+
+	out, err := Carrier(fmt.Sprintf("push %s --verbosity 1", appName), appDir)
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	// And check presence
+
+	out, err = Carrier("apps", "")
+	Expect(err).ToNot(HaveOccurred(), out)
+	Expect(out).To(MatchRegexp(appName + `.*\|.*1\/1.*\|.*`))
+}
+
+func makeCustomService(serviceName string) {
+	out, err := Carrier(fmt.Sprintf("create-custom-service %s username carrier-user", serviceName), "")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	// And check presence
+
+	out, err = Carrier("services", "")
+	Expect(err).ToNot(HaveOccurred(), out)
+	Expect(out).To(MatchRegexp(serviceName))
+}
+
+func makeCatalogService(serviceName string) {
+	out, err := Carrier(fmt.Sprintf("create-service %s mariadb 10-3-22", serviceName), "")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	// And check presence
+
+	out, err = Carrier("services", "")
+	Expect(err).ToNot(HaveOccurred(), out)
+	Expect(out).To(MatchRegexp(serviceName))
+}
+
+func bindAppService(appName, serviceName, org string) {
+	out, err := Carrier(fmt.Sprintf("bind-service %s %s", serviceName, appName), "")
+	Expect(err).ToNot(HaveOccurred(), out)
+
+	// And check deep into the kube structures
+	verifyAppServiceBound(appName, serviceName, org)
+}
+
+func verifyAppServiceBound(appName, serviceName, org string) {
+	out, err := helpers.Kubectl(fmt.Sprintf("get deployment -n carrier-workloads %s.%s -o=jsonpath='{.spec.template.spec.volumes}'", org, appName))
+	Expect(err).ToNot(HaveOccurred(), out)
+	Expect(out).To(MatchRegexp(serviceName))
+
+	out, err = helpers.Kubectl(fmt.Sprintf("get deployment -n carrier-workloads %s.%s -o=jsonpath='{.spec.template.spec.containers[0].volumeMounts}'", org, appName))
+	Expect(err).ToNot(HaveOccurred(), out)
+	Expect(out).To(MatchRegexp("/services/" + serviceName))
+}
+
+func deleteApp(appName string) {
+	out, err := Carrier("delete "+appName, "")
+	Expect(err).ToNot(HaveOccurred(), out)
```

Comment:
we use these  `delete*` methods on AfterEach blocks as cleanups. Sometimes when the test fails, one of these cleanups fail (e.g. trying to delete an app when the tests failed to create one). This make the AfterEach block to fail early and skip the rest of the cleanups. That's why I made them to not panic here: https://github.com/SUSE/carrier/commit/0c472ce52d7c8c9fd4b9b54b52a34c2421119143

so that cleanup can continue. If we can't to keep this functionality, we should probably return an error from these methods here, and run the `Expect(err).ToNot(HaveOccurred())` only where needed.

Reply:
I believe I saw these changes yesterday. And today I clearly missed them in my conflict resolution. Will fix.

Reply:
Looking over them ... We have two different "delete service" ... One with the 10m timeout, and then those which should not fail because they are cleanup ... Likely similar for "delete app" ... I will separate these two uses n the utilities, rewrite the after blocks, and then see if both forms are still needed.

Reply:
Done with commit ad4b4b67fa48464f0e5b8e2624ae74342d2034fc